### DISCUSSION TITLE
Community: use case ideas

### DISCUSSION THREAD

**Discussion by raphodn** [openfoodfacts/open-prices#590] [Ideas]
### A list of use case ideas

See below

### Additional info

- known reuses: https://prices.openfoodfacts.org/community
- https://github.com/openfoodfacts/open-prices/discussions/562

**Comment by raphodn** [comment]
### Price of Nutella

A map displaying all the known Nutella prices
Need to list the different barcodes (Nutella come in different sizes)

cc @aleene

**Comment by raphodn** [comment]
### Who sells avocados from Peru ?

A simple website to point to shops that sell exotic/far-away fruits & vegetables, to highlight the environmental impact/absurd

#### Examples of "pre-filled" searchs

|Example|Details|
|---|---|
|pomme venant de l'hemisphère sud (e.g. Argentine, Afrique du Sud, Chili)|été = pas la saison (récolte débute fin aout-mioctobre, vide mai-juin, juillet/aout)|
|avocat extra-européen|peut être produit + localement, et il y a une saison|
|||

Plusieurs types "d'absurdes"
- pas la saison (on pourrait trouver local)
- ca vient de très loin

**Comment by serpico** [comment]
There was something about the price per 100g of proteins in the 31st July 2024's meeting minutes ( including a screenshot of the terminal running the tool ) but no link to a repo ( maybe it was the way the user wanted it...)

**Reply by raphodn** [reply]
@albertaillet

**Reply by albertaillet** [reply]
yes, this is a tool that I am working on!

**Reply by serpico** [reply]
@albertaillet Hi Albert, would you be ok pointing to your tool's repo from somewhere in the open prices repo/website ? In order for passers-by/users to know about it. I read about it in the minutes of a weekly meeting ( a few weeks/months back ) but I felt it was buried there, almost indiscoverable :)

**Comment by serpico** [comment]
Maybe I should close #589 and add it here ?

**Reply by raphodn** [reply]
we can have both for now, and see if someone picks it up !
(as a functionnality inside the web app, or as a simple third-party app :)
